drivers/media/i2c/arducam_64mp.c

```diff
@@ -2291,6 +2297,19 @@ static int arducam_64mp_init_controls(struct arducam_64mp *arducam_64mp)
 						     ARDUCAM_64MP_PIXEL_RATE,
 						     ARDUCAM_64MP_PIXEL_RATE, 1,
 						     ARDUCAM_64MP_PIXEL_RATE);
+	if (arducam_64mp->pixel_rate)
+		arducam_64mp->pixel_rate->flags |= V4L2_CTRL_FLAG_READ_ONLY;
```

Comment:
This happens automatically due to the core - https://elixir.bootlin.com/linux/latest/source/drivers/media/v4l2-core/v4l2-ctrls-defs.c#L1512

Reply:
Got it, I was wondering why the read-only flag wasn't added before, but recently imx477 had the same commit, so I guess it's redundant too.